### PR TITLE
chore(docs): improve documentation for Azure debugging

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

This pr adds info about azure set-up and debugging in VSCode 

### Description

Readme files are modified

Feat @puchy22 

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/getting-started/requirements.md [hunk 1]
```diff
@@ -71,14 +71,23 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
...
+1. Access to Microsoft Entra ID
+2. Go to "App registrations" and register a new application
+  ![Register an Application page](../img/register-application.png)
+3. Once your application is registered, select "API permissions" blade
+4. Add the needed permissions:
+    - `Directory.Read.All`
+    - `Policy.Read.All`
+
+    ![EntraID Permissions](../img/AAD-permissions.png)
+
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
1. Access to Microsoft Entra ID
2. In the left menu bar, go to "App registrations"
3. Once there, in the menu bar click on "+ New registration" to register a new application
4. Fill the "Name, select the "Supported account types" and click on "Register. You will be redirected to the applications page.
  ![Register an Application page](../img/register-application.png)
4. Select the new application
5. In the left menu bar, select "API permissions"
6. Then click on "+ Add a permission" and select "Microsoft Graph"
7. Once in the "Microsoft Graph" view, select "Application permissions"
8. Finally, search for "Directory" and "Policy" and select the following permissions:
    - `Directory.Read.All`
    - `Policy.Read.All`

    ![EntraID Permissions](../img/AAD-permissions.png)

```

File: docs/getting-started/requirements.md [hunk 2]
```diff
@@ -87,6 +96,19 @@ Regarding the subscription scope, Prowler by default scans all the subscriptions
...
+To assign this roles, follow the instructions:
+
+1. Access your subscription, then select your subscription.
+2. Select "Access control (IAM)".
+3. In the overview, select "Roles"
+  ![IAM Page](../img/page-IAM.png)
+4. Click on Add and select 'Add role assigment'
+5. In the bar, search `Security Reader` and select it.
+6. In the Members blade, select a member.
+7. Review + assign to apply the new role.
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
1. Access your subscription, then select your subscription.
2. Select "Access control (IAM)".
3. In the overview, select "Roles"
  ![IAM Page](../img/page-IAM.png)
4. Click on "+ Add" and select "Add role assignment"
5. In the search bar, type `Security Reader`, select it and click on "Next"
6. In the Members tab, click on "+ Select members" and add the members you want to assign this role.
7. Click on "Review + assign" to apply the new role.
```

File: docs/getting-started/requirements.md [hunk 3]
```diff
@@ -87,6 +96,19 @@ Regarding the subscription scope, Prowler by default scans all the subscriptions
...
+
+*Repeat these steps for `Reader`role*
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
*Repeat these steps for `Reader` role*
```

File: docs/getting-started/requirements.md [hunk 4]
```diff
@@ -71,14 +71,23 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
...
+  ![Register an Application page](../img/register-application.png)
+3. Once your application is registered, select "API permissions" blade
+4. Add the needed permissions:
+    - `Directory.Read.All`
+    - `Policy.Read.All`
+
+    ![EntraID Permissions](../img/AAD-permissions.png)
+
 
 #### Subscriptions scope
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md:97]: Review the double space after `subscription`

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: docs/getting-started/requirements.md (hunk 1 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md]: Done

File: docs/getting-started/requirements.md (hunk 2 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md]: Done

File: docs/getting-started/requirements.md (hunk 3 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md]: Done

File: docs/getting-started/requirements.md (hunk 4 above)
Comment (pedrooot) [reply docs/getting-started/requirements.md:97]: Done

**Review by jfagoagas — APPROVED** [review]
Awesome! Thanks for improving our docs!!

**Pull request merged**